### PR TITLE
Add manual testing cases for DG

### PR DESCRIPTION
**Pull request by SherniceSng**
did this for view, delete, vieww and deletew

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -499,63 +499,75 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case: `view x` (where x is larger than the size of person list)<br>
+       Expected: Error message prompting the user to choose an index within the range shown.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+    1. Other incorrect view commands to try: `view`, `view x` (where x is a negative integer)<br>
+       Expected: Similar to point #1(iii).
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+    1. Test case: `view Alice Tan` <br>
```
Comment (VedJoshi) [inline docs/DeveloperGuide.md:526]: good manual test

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -594,6 +606,79 @@ testers are expected to do more *exploratory* testing.
...
+    1. Other incorrect vieww commands to try: `vieww`, `vieww x` (where x is a negative integer)<br>
+       Expected: Similar to point #1(iii).
+
+    1. Test case: `vieww Alice` <br>
+       Expected (Unique Alice): The wedding of `Alice` will be shown. Success action will be carried out for that wedding.<br>
+       Expected (Duplicated Alice): Weddings with name field matching `Alice` exactly will be shown.
+       Message will be shown to prompt the user to specify which wedding of `Alice` they want to view.<br>
+       Expected (No Alice): No wedding is viewed. Error details is shown in the status message.
+
+1. Viewing a wedding while a filtered list of weddings is shown
```
Comment (VedJoshi) [inline docs/DeveloperGuide.md:636]: I am not quite sure what you mean by this

**Review by VedJoshi — COMMENTED** [review]
LGTM  - unsure of one part

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (SherniceSng) [reply docs/DeveloperGuide.md]: It's for cases when say there are two weddings with the same name (e.g. Alice). If I `vieww anna`,  the filtered list of anna weddings will appear and we will need to specify the index. The index will be according to the filtered list of wedding shown. Similar to the normal view command.

Does that make sense? How should I phrase it to make it less confusing?

**Pull request merged**